src/Frontend/Testing.hs

```diff
@@ -73,10 +74,12 @@ type RenderHtmlSource' =
        "idea"  :> Capture "iid" (AUID Idea)  :> Get '[PlainText] String
   :<|> "topic" :> Capture "tid" (AUID Topic) :> Get '[PlainText] String
```

Comment:
I know it's unrelated to your PR but it's Misc...

These captures can be `Idea ::>` and `Topic ::>`